cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2754,6 +2754,83 @@ action.cyences_send_email_action = 1
 
 
+[Email - Suspicious Subject or Attachment]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 56 * * * *
+description = This alert will trigger when suspicious attachment or subject found in mail. \
+NOTE: cs_email_user_domain macro must be configured with the domain name in order to use this alert. \
+Data Collection - Gmail data (Splunk Add-on for Google Workspace). 
+dispatch.earliest_time = -1d@h
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_gsuite` sourcetype="gws:gmail" _index_earliest=-61m@m _index_latest=-1m@m \
+| rename "message_info.attachment{}.file_extension_type" as file_extension_type, "message_info.num_message_attachments" as num_message_attachments, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject \
+| search `cs_email_suspicious_attachment_extension` OR `cs_email_suspicious_subjects_and_title(Subject)` \
+| stats count, min(_time) as firstTime, max(_time) as lastTime, values(file_extension_type) as file_extension_type, values(message_info.payload_size) as payload_size, values(src_user_domain) as source_domain, values(recipient_domain) as dest_domain by SenderAddress, RecipientAddress, Subject, num_message_attachments \
+|  eval cyences_severity = if(not source_domain=`cs_email_user_domain` and dest_domain=`cs_email_user_domain`, "medium", "low")
+| `cs_human_readable_time_format(firstTime)` \
+| `cs_human_readable_time_format(lastTime)` \
+| sort - count \
+| `cs_email_suspicious_attachment_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_email_suspicious_attachment_filter
+action.cyences_notable_event_action.contributing_events = `cs_gsuite` sourcetype="gws:gmail" | rename "message_info.attachment{}.file_extension_type" as file_extension_type, "message_info.num_message_attachments" as num_message_attachments, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject | search `cs_email_suspicious_attachment_extension` OR `cs_email_suspicious_subjects_and_title(Subject)`
+action.cyences_notable_event_action.system_compromised_search = | stats count by SenderAddress, RecipientAddress, Subject
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_gsuite` sourcetype="gws:gmail" | rename "message_info.attachment{}.file_extension_type" as file_extension_type, "message_info.num_message_attachments" as num_message_attachments, recipient as RecipientAddress, src_user as SenderAddress, subject as Subject | search `cs_email_suspicious_attachment_extension` OR `cs_email_suspicious_subjects_and_title(Subject)` SenderAddress=$row.SenderAddress$ RecipientAddress=$row.RecipientAddress$ Subject=$row.Subject$
```

Comment:
Good idea to use props.conf for renames, for example

"message_info.attachment{}.file_extension_type" as attachement_file_extension_type